### DISCUSSION TITLE
A test question with sourdough

### DISCUSSION THREAD

**Discussion by spdegabrielle** [racket/typed-racket#1005] [Q&A] [answered]
Testing testing I wanted to test the q&a functionality.
![C30934F8-A7C3-4837-B066-8ADA4ED1C3C0](https://user-images.githubusercontent.com/108047/101682066-06799b80-3a5b-11eb-9107-1a3e27de74ba.jpeg)

**Comment by spdegabrielle** [comment]


**Reply by spdegabrielle** [reply]
Yes

**Comment by spdegabrielle** [comment]


**Reply by spdegabrielle** [reply]
No!

**Comment by jbclements** [comment, marked as answer]
It looks like you're not getting great oven spring on those... :)

**Reply by byu** [reply]
Was the sourdough scored before baking? Scoring can help with greater oven spring. :)

**Reply by spdegabrielle** [reply]
no it wasn't scored.
